http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -882,7 +892,7 @@ paths:
                 $ref: "#/components/schemas/ErrorReport"
 
   "/api/v2/runtimes/{workspaceId}/deleteAll":
-    post:
+    delete:
```

Comment:
We debated on this for a bit, but it is intentionally POST. The thought is `delete` is an HTTP verb on a specific resource. Since `deleteAll` does not specify a resource (and could even result in nothing being deleted with a successful status code if no resources exist), `POST` best describes the action this endpoint performs (generic side-effectful update of state)

Reply:
ohhh ok, I honestly thought it was a mistake, but I should've checked with you guys, that makes sense!